tests/SprykerEcoTest/Zed/PunchoutCatalogs/Business/PunchoutCatalogsFacadeTest.php

```diff
@@ -82,14 +86,16 @@ public function testFindConnectionByIdRetrievesConnectionWhenItExists(): void
     public function testFindConnectionByIdRetrievesPasswordFormVaultWhenItExists(): void
     {
         // Arrange
+        $companyBusinessUnitTransfer = $this->tester->createCompanyBusinessUnit();
         $idPunchoutCatalogConnection = $this->tester->havePunchoutCatalogConnection([
```

Comment:
Can you please move the `PunchoutCatalogConnection` transfer creation to the separate method, which will accept `CompanyBusinessUnitTransfer`?